src/lib/components/SectionListItem.svelte

```diff
@@ -25,9 +25,7 @@
 		<a
 			class="section-list-item__a"
 			data-testid={isSession ? 'session-item' : 'knowledge-item'}
-			aria-label={(isSession
-				? $i18n.t('sessions', { count: 1 })
-				: $i18n.t('knowledge', { count: 1 })) + `: ${id}`}
+			aria-label={(isSession ? $LL.session() : $LL.knowledge()) + `: ${id}`}
```

Comment:
We only really had one pluralized string: `Session/s`, while the library does [support pluralization](https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/runtime#plural), I ended it up just adding dedicated keys for both versions.